### DISCUSSION TITLE
Visualizing PMTiles with Leafmap and MapLibre

### DISCUSSION THREAD

**Discussion by giswqs** [protomaps/PMTiles#414] [Show and tell]
Leafmap now supports interactive visualization of PMTiles using #MapLibre 🗺️ This enables seamless exploration of large vector datasets in serverless mapping applications. The demo showcases visualizing a global building dataset (~200 GB) stored as PMTiles, and provides interactive controls to adjust visibility, opacity, and color 🌎

- GitHub: https://github.com/opengeos/leafmap
- Notebook: https://leafmap.org/notebooks/93_maplibre_pmtiles

https://github.com/protomaps/PMTiles/assets/5016453/8bac56ce-977d-46bd-9e3b-ea034285794b

**Comment by nvkelso** [comment]
Rad!
